integration_tests/setup_test.go

```diff
@@ -43,8 +43,8 @@ import (
 )
 
 const (
-	testDenom           = "testsomm"
-	initBalanceStr      = "110000000000testsomm"
+	testDenom           = "usomm"
```

Comment:
Alternatively to the `BaseCoinUnit` thing we're also defining a constant here for the testing env. Either way, we can use one hard reference or the other.